containers/python-3-device-simulator-express/.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "/opt/vscode/extensions/ms-python.devicesimulatorexpress/venv/bin/python"
```

Comment:
Not sure if this is required, but FYI, this won't be picked up when added via VS Code, so if this is needed to run, this should be moved as the python.pythonPath in the `settings` section of `devcontainer.json`.